### ISSUE TITLE
Cullface not properly rendered on translucent block (MC Patcher's Better Glass)

### ISSUE THREAD

**Issue by MaxiKevey** [jellysquid3/Sodium#1573]
### Version information

mc1.19.2-0.4.4+build.18

### Expected Behavior

I'm currently in the process of trying to make a texture pack compatible with Fabric which was previously depending on Optifine. Where I now come across this issue. I'm new to Fabric and Sodium, so I don't really know if this has something to do with Sodium or Continuity. But I gotta start somewhere. So initially what should be happening is that in the picture. Rendered backfaces through the block itself. This works to an extend, but only when looking through the block on the northern side.
![2023-01-11_23 22 32](https://user-images.githubusercontent.com/28632262/211930755-8237aa10-5271-4690-96cd-96b224a5189d.png)

### Actual Behavior

Instead however when viewing from all other sides some faces are not rendered.

![2023-01-11_23 23 15](https://user-images.githubusercontent.com/28632262/211932946-57d20a82-d4d4-4c42-9593-0657d835b6be.png)

### Reproduction Steps

It's a bit convoluted to explain, so the best way to see how this is achieved is either by downloadig Misa's Texture Pack or a smaller version where I deleted all of the other textures under this link: [https://www.mediafire.com/file/1n4jbg9obnyvit0/Misa_Glass.zip/file](url)
Basically the block model has all it's faces as seperate cubes which have an empty texture on the inside which is later replaced with matchTiles to get CTM to work.

### Java version

17.0.3

### CPU

Intel Core i7-5820K @3.99 GHz

### GPU

Nvidia GeForce RTX 3080 Ti

### Additional information

Other mods used to get this working:

- continuity-2.0.1+1.19
- fabric-api-0.72.0+1.19.2
- indium-1.0.9+mc1.19.2
- lithium-fabric-mc1.19.2-0.10.4

**Comment by jellysquid3** [comment]
This should no longer be an issue in Sodium 0.6.0.
